src/components/Markdown/index.js

```diff
@@ -0,0 +1,7 @@
+import ReactMarkdown from 'react-markdown'
+
+import React from 'react'
+
+export default ({children}) => (
```

Comment:
Is the purpose of using a wrapper component just to explicitly state that we use a `Markdown` component? As opposed to just using the `ReactMarkdown` component in `LessonData`?